frontend/src/scenes/billing/billingProductLogic.ts

```diff
@@ -349,6 +367,35 @@ export const billingProductLogic = kea<billingProductLogicType>([
                 redirectPath && `&redirect_path=${redirectPath}`
             }`
         },
+        activateTrial: async () => {
```

Comment:
Just to be certain people get the features - did you make sure this updates the availableFeatures array after it succeeds? And same but opposite for when they cancel.

Reply:
Yep, will do more testing but it reloads billing after the call.